### ISSUE TITLE
Define website footer

### ISSUE THREAD

**Issue by kraulain** [FreeJavaBootcamp/fjb-website#3]
### Background

We need to update the current site footer to contain up-to-date information about the site and shortcut links.

---

### Requirements

- Update the footer to contain up-to-date info

---

### Resources
 
- First and second item on [Bootstrap footer example](https://getbootstrap.com/docs/5.2/examples/footers/)
- [JBake Documentation](https://jbake.org/docs/2.6.7/)

**Comment by Wandji69** [comment]
@marrious11 are you still working on this?
Also, which of the footer design are we using here?

**Comment by marrious11** [comment]
Yes oh papa still on it. Did you want us to work on it together @Wandji69 ?
 I am hoping that this one is a better fit
![image](https://github.com/FreeJavaBootcamp/fjb-website/assets/66856309/6985246b-4a22-468d-9d20-ecd14b31ec31)

,since we'll need to update the footer to have the latest info about the website and shortcut links

**Comment by kraulain** [comment]
@marrious11 Please note that the task description recommended using the first footer example from https://getbootstrap.com/docs/5.2/examples/footers/

This is because it is simple enough and will enable us to release the site sooner.
